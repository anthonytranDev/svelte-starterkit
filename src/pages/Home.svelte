<h1 data-testid="Home">Home</h1>
