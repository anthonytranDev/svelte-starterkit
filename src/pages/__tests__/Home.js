import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/svelte"

import Home from "../Home.svelte"

describe("Home", () => {
  it("renders", () => {
    const { getByTestId } = render(Home)

    expect(getByTestId("Home")).toMatchSnapshot()
  })
})
