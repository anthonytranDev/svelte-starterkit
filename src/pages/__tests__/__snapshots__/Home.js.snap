// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<h1
  data-testid="Home"
>
  Home
</h1>
`;
